api/ttn/lorawan/v3/identityserver.proto

```diff
@@ -70,6 +70,9 @@ message IsConfiguration {
 
     message ContactInfoValidation {
       google.protobuf.BoolValue required = 1;
+      google.protobuf.Duration token_ttl = 2;
+      // RetryInterval is the minimum interval for resending a validation email
```

Comment:
The comments don't have to be in the same format that we have in Go - in general in the protos we use something like `The minimum interval between validation emails.`